baiji/util/tempfile.py

```diff
@@ -7,10 +7,13 @@
 '''
 from __future__ import absolute_import
 import os
+from baiji.util.shutillib import mkdir_p
 
 def _override_tmp(fn, args, kwargs):
     if 'dir' not in kwargs:
         kwargs['dir'] = os.environ.get('BAIJI_TMP', None)
+        if kwargs['dir'] is not None:
+            mkdir_p(kwargs['dir'])
```

Comment:
I get that part but I was under the impression that the `_override_tmp` function exists because `mkstmp` on some systems was failing to do what it is supposed to, namely creating this local temp dir.